lib/galaxy/tool_util/verify/__init__.py

```diff
@@ -194,6 +193,20 @@ def _verify_checksum(data, checksum_type, expected_checksum_value):
         raise AssertionError(message)
 
 
+def files_delta(file1, file2, attributes=None):
+    """Check the contents of 2 files for size differences."""
+    if attributes is None:
+        attributes = {}
+    delta = int(attributes.get('delta', DEFAULT_DELTA))
+    delta_frac = float(attributes.get('delta_frac', DEFAULT_DELTA_FRAC))
+    s1 = os.path.getsize(file1)
+    s2 = os.path.getsize(file2)
+    if abs(s1 - s2) > delta:
+        raise AssertionError('Files %s=%db but %s=%db - compare by size (delta=%s) failed' % (file1, s1, file2, s2, delta))
+    if not (s1 - (s1 * delta_frac) <= s2 <= s1 + (s1 * delta_frac)):
```

Comment:
```suggestion
    if delta_frace is not None and not (s1 - (s1 * delta_frac) <= s2 <= s1 + (s1 * delta_frac)):
```

Reply:
I see. I will go for it. Can you edit your suggestion `frace` -> `frac`.

Reply:
Sure, fixed this one and the one above.